### PR TITLE
Update .babelrc

### PR DESCRIPTION
**Pull request by hankolsen**
First of all, thank you for a great project! I just started playing around with it and it is really nice!

The problem:
I'm using honeycomb in a create-react-app project and everything worked fine until I tried to do a production build. 
Running `yarn build` in the create-react-app project resulted in an error:
`Failed to minify the code from this file: ./node_modules/honeycomb-grid/dist/honeycomb.esm.min.js:1:764`

After some debugging I noticed that the error came from a few lines in the code that used template literals. Those lines were not transpiled to concatenated strings. A bit of experimenting with different `.bablerc` settings later, the error from create-react-app build was gone.

So, my proposed solution (to an issue that no one else seems to have had...), is to add "ie >= 11" to the browsers list in .babelrc. That change transpiled the template literals to concatenated strings, and the create-react-app can handle the minification during it's build process.

### CODE REVIEW

**Comment by flauwekeul** [comment]
Thanks for the PR 🙂Just curious: what are you using honeycomb for?

**TL;DR**: I'll merge your PR 👍

**Background**: apparently, your bundler (webpack) assumes dependencies of your project (honeycomb-grid) are in ES5 (or at least whatever IE11 can handle). I thought it would use the ES2015 module version of the code, so it can determine which parts of honeycomb it needs...

I want to change how the package is built, currently I only offer minified versions of the code. But it seems wise to also offer unminified versions. But strangely, rollup (the bundler I use) doesn't offer an easy way to make minified and unminified versions of the code. Which seems like it might be an unusual practice... Anyway, I'll figure it out one day 😅

**Comment by flauwekeul** [comment]
Oops...Multiple tests fail (I should setup a CI tool...) and I remember why I didn't transpile for older browsers: I use some features that can't be correctly transpiled to ES5 (like extending `Array`).

We'll have to look for another solution to your problem 😢

**Comment by hankolsen** [comment]
Aah, I just noticed the test failures too, sorry about that.

I forgot to mention that the error message from the create-react-app build script contained this link [http://bit.ly/2tRViJ9]( http://bit.ly/2tRViJ9), where it basically says what you summarized regarding ES5.

And to answer your question, I'm playing around trying to create a way to visualize data in a hexgrid. I tried some d3js stuff, but found your project much more intuitive.
